### ISSUE TITLE
Several ideas to implement

### ISSUE THREAD

**Issue by ops1ops** [import-js/eslint-plugin-import#2476]
Hello, i want files ending with ./css to be LAST in sibling group, so to my mind there are 2 potential features:
1. In pathGroups you need to add property disabling newlines-between for this pathGroups
2. Add order property or something like this "order": ["begin" | "end"]

Please implement one of these features. Or if there are workaround for my situation, share please.

By the way adding "name" property for pathGroups and then being able to order other pathGroups relatevily to named pathGroups would be also cool

**Comment by ehoogeveen-medweb** [comment]
Another suggestion: Allow certain groups or pathGroups to be exempt from sorting.

I generally want my imports to be sorted alphabetically by import path, but for external libraries there might be side effects that need to happen in a specific order (I actually only use these import statements for mouseover information and autocomplete in VSCode, a post-processing step turns them into unassigned imports).

**Comment by ljharb** [comment]
@ehoogeveen-medweb any import with side effects should omit all bindings (like `import 'foo';`). Any bindingless import is assumed to have side effects, and thus, will *never* have anything before it moved after, or anything after it moved before.

**Comment by ehoogeveen-medweb** [comment]
@ljharb I know, but that's not what I'm talking about. I wrote a babel plugin to take something like
```js
import $ from 'ext/jquery';
import moment from 'ext/moment';

import { mySymbol } from 'myFile';
```
and turn it into something like
```js
import 'base/path/ext/jquery.js';
import 'base/path/ext/moment.js';

import { mySymbol } from 'base/path/myFile.js';
```
so that I can get autocomplete and mouseover information in VSCode while ultimately still using UMD globals (eventually I'd like to modernize our code completely but that's still a long way away).

So from the perspective of `import/order` these imports are not unassigned which means they can be reordered, but in reality the ordering might matter. Certainly a bit of a niche case, but I think "sort these imports, but not these ones" would be a nice feature to have :)

**Comment by ljharb** [comment]
That seems like it’d create a lot of confusion for developers, in addition to breaking anyone that uses a moment or $ identifier for something else, or uses different identifiers for moment or jquery.

**Comment by ehoogeveen-medweb** [comment]
We're coming from a situation where these things just magically existed and the fact that they did was completely undocumented, so defining them in a way where VSCode will tell you what it's supposed to be and ESLint will warn you if you try to redefine it is a major step up. The hard part is the initial step of documenting what comes from where. I'm not advocating it as a solution for other projects, but it's a major improvement of the development experience for us.
